Projects/Foundations/Schemas/Docs-Topics/D/Divergences/Divergences-Data-Mine/Divergences-data-mine-003-commodity-channel-index-indicator.json

```diff
@@ -1,74 +0,0 @@
-{
-    "topic": "Divergences Data Mine",
-    "pageNumber": 3,
-    "type": "Commodity Channel Index Indicator - CCI",
-    "definition": {
-        "text": "This indicator produces the Commodity Channel Index oscillator, as originally published by Donald Lambert in 1980."
-    },
-    "paragraphs": [
-        {
```

Comment:
Will there be replacement for this page?

Reply:
I'm trying to consolidate the files as there's been some duplicates added here from SA when the translation was added, which is confusing a few things now!

Reply:
So you are sure this file should get deleted?

Reply:
yeah I've replaced it with the file: divergences-data-mine-003-commodity-channel-index-indicator-cci.json